eval/asserts/coding-review-metrics.test.mjs

```
@@ -0,0 +1,71 @@
+import assert from 'node:assert/strict';
+import { readFile } from 'node:fs/promises';
+import test from 'node:test';
+import { parse } from 'yaml';
+
+const configUrl = new URL('../promptfooconfig.coding.yaml', import.meta.url);
```

ESLint

[error] 'URL' is not defined.

(no-undef)